main/src/main/java/org/mobilitydata/gtfsvalidator/cli/Arguments.java

```diff
@@ -77,6 +77,11 @@
       help = true)
   private boolean help = false;
 
+  @Parameter(
+      names = {"-p", "--pretty"},
+      description = "Pretty json output")
+  private Boolean pretty = false;
```

Comment:
boolean - starting with lowercase.

I see that you systematically write Boolean with capital. Do you know why?

Reply:
A very bad habit of mine - I'll keep an eye open to not reproduce this in the future. Thanks Alexei!